project/src/main/kotlin/org/ionproject/core/calendar/CalendarRepository.kt

```diff
@@ -0,0 +1,36 @@
+package org.ionproject.core.calendar
+
+import org.ionproject.core.calendar.icalendar.*
+import org.ionproject.core.calendar.icalendar.types.*
+
+class CalendarRepository {
```

Comment:
Declare an interface, for ease of testing.
`
interface CalendarRepository
@Component class CalendarRepositoryImpl
`
according to the conventions used. The unit tests will then tell the Spring runtime to inject a higher priority Repository implementation.